internal/metrics/delivery/handlers.go

```diff
@@ -0,0 +1,268 @@
+package delivery
+
+import (
+	"encoding/json"
+	"github.com/MaxBoych/MetricsService/internal/metrics"
+	"github.com/MaxBoych/MetricsService/internal/metrics/models"
+	"github.com/MaxBoych/MetricsService/pkg/logger"
+	"github.com/go-chi/chi/v5"
+	"go.uber.org/zap"
+	"net/http"
+	"strconv"
+)
+
+type MetricsHandler struct {
+	useCase metrics.UseCase
+}
+
+func NotFound(w http.ResponseWriter, r *http.Request) {
```

Comment:
```suggestion
func NotFound(w http.ResponseWriter, _ *http.Request) {
```
если сигнатура должна соблюдаться а переменная не нужна - экономими память , не выдеяем ее